fitbenchmarking/fitbenchmarking
# Removing reduntent `test_main.py`

#### Description of Work

Fixes #532 

Removes `test_main.py` which are a duplicate of the system tests.

#### Testing Instructions

1. Check nothing else is removed

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
